src/editor/components/menu/editor-toolbar-menu.jsx

```diff
@@ -5,19 +5,23 @@ import PropTypes from "prop-types";
 import NotebookIconMenu from "./icon-menu";
 import tasks from "../../user-tasks/task-definitions";
 import NotebookMenuItem from "./notebook-menu-item";
-import { connectionModeIsServer } from "../../tools/server-tools";
+import {
+  connectionModeIsServer,
+  notebookIsATrial
+} from "../../tools/server-tools";
 
 export class EditorToolbarMenuUnconnected extends React.Component {
   static propTypes = {
     isServer: PropTypes.bool.isRequired,
-    canUpload: PropTypes.bool
+    canUpload: PropTypes.bool,
+    isTrialNotebook: PropTypes.bool
   };
 
   render() {
     return (
       <NotebookIconMenu>
         {this.props.isServer && <NotebookMenuItem task={tasks.newNotebook} />}
-        {this.props.isServer && (
+        {!this.props.isTrialNotebook && (
           <NotebookMenuItem task={tasks.toggleHistoryModal} />
```

Comment:
I think I'd prefer to disable the item in this case, since there is a way to get access to the functionality (log in). Try this instead:

```jsx
<NotebookMenuItem task={tasks.toggleHistoryModal} disabled={this.props.isTrialNotebook} />
```

Reply:
Thanks, good point! Total React n00b - it seems the above code won't work either since the item doesn't get rerendered on login? I'll try to figure out how this actually works.

Reply:
I think the reason is that the notebook tryit mode boolean isn't set to false after login and notebook creation - PR updated, hope this works!